src/components/modalDialog/GenericModalDialog.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from 'react';
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from "@mui/material"
+
+export interface ModalDialogProps {
+    isOpen: boolean;
+    onClose: () => void;
+    onCancel: () => void;
+    title: string;
+    subtitle: string;
+    children?: JSX.Element;
+  }
+
+export const GenericModalDialog: FC<ModalDialogProps> = ({
+    isOpen,
+    onClose,
+    onCancel,
+    title,
+    subtitle,
+    children
+  }) => {
+ 
+    const handleClose = (
+      event: {},
+      reason: "backdropClick" | "escapeKeyDown"
+    ) => {
+      if (reason !== "backdropClick") {
+        onClose();
+      }
```

Comment:
¿Puede ser que esto sea error de lógica?
Acá debe triggerearse la cancelación.

```suggestion
      if (reason !== "backdropClick") {
        onCancel();
      }
```

Reply:
si si, es cierto. El onClose x el onCancel es asi, si.